### PR TITLE
add MonoFocus v1.0.beta35

### PR DESCRIPTION
**Pull request by 0xdevalias**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask-versions/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

## See Also

- https://monofocus.app/
  - https://monofocus.app/thank-you

cc // FYI MonoFocus devs: @bartekjacak @neonowy @wowu

### CODE REVIEW

**Comment by miccal** [comment]
Thank you for your contribution to `homebrew-cask-versions` @0xdevalias.

Could you please try to add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap), thanks.

**Comment by 0xdevalias** [comment]
> Could you please try to add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap), thanks.

@miccal I tried running the `brew createzap` program that was suggested in a previous PR, and it didn't seem to produce anything to be included; so I assumed there was nothing required for it.

File: Casks/monofocus.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  sha256 :no_check
+
+  url "https://updates.monofocus.app/MonoFocus.latest.dmg"
+  name "MonoFocus"
+  desc "Keep all tasks from your todo apps on your menu bar"
+  homepage "https://monofocus.app/"
+
+  livecheck do
+    url :url
+    strategy :extract_plist
```
Comment (samford) [inline Casks/monofocus.rb:12]:
```suggestion
    regex(/^v?(\d+(?:\.\d+)+\S*)/i)
    strategy :extract_plist do |items, regex|
      items["com.zbudniewek.WorkingOn"]&.short_version&.scan(regex)&.map { |match| match[0] }
    end
```

Looks like we'll need to extract the version from the `short_version` text (e.g., `1.0.beta35 (32cc9073)`).

**Comment by miccal** [comment]
Thank you @samford.

Thank you @0xdevalias -- since there is no stable version at the moment, this Cask should be submitted to the main `homebrew-cask` repository as explained [here](https://docs.brew.sh/Acceptable-Casks#stable-versions).

I am going to go ahead a close this PR and resubmit this to the main repository on your behalf.

**Pull request closed without merging**